docs/network/monitoring.md

```diff
@@ -0,0 +1,314 @@
+
+## Metrics
+
+Metrics for the node are always available on the ``/metrics`` endpoint of the address and port of the ``--rpc-addr`` argument. One can query this using:
+
+```bash
+curl http://<rpc_addr>:<rpc_port>/metrics
+```
+
+You will be presented with the list of defined and computed metrics as follows:
+
+```
+#HELP metric description
+#TYPE metric type
+octez_metric_name{label_name=label_value} x.x
+```
+
+Each exported metric has the following form:
+
+```
+octez_evm_node_metric{label_name=label_value;...} value
+```
+
+Each metric name starts with ``octez`` as its namespace, followed by the a subsystem name ``evm_node``, and the name of the metric.
+It follows the [OpenMetrics specification](https://openmetrics.io/).
+
+A metric may provide labeled parameters which allow for different instances of the metric, with different label values.
+For instance, the metric ``octez_evm_node_calls_method`` has a label name ``method`` which allows this metric to have one value for each kind of method.
+
+```
+#HELP octez_evm_node_calls_method Method call counts
+#TYPE octez_evm_node_calls_method counter
+octez_evm_node_calls_method{method="debug_traceTransaction"} 13594.000000
+octez_evm_node_calls_method{method="eth_blockNumber"} 2.000000
+...
+```
+
+Metrics provide information about the node in the form of a [gauge](https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#gauge) that can increase or decrease (like the base gas price of a block),
+a [counter](https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#counter) that can only increase (like the number of calls to a particular method),
+or a [histogram](https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#histogram) used to track the size of events and how long they usually take (e.g., the time spent processing a block).
+
+### Metrics listing
+
+Some of the metrics are computed when scraped from the node. As there is no rate limiter, you should consider scraping wisely and adding a proxy for a public endpoint, to limit the impact on performance.
+
+The metric names in the table omit the namespace and subsystem for readability, so to have the full metric name you must add `octez_evm_node_`, e.g. `octez_evm_node_boostrapping`.
+
+|             Metric names       |    Type   |                                                       Description                                                      |    Relevant mode    |
+|:------------------------------:|:---------:|:----------------------------------------------------------------------------------------------------------------------:|:-------------------:|
+| bootstrapping                  | gauge     | 1.0 if the EVM node is catching up with its upstream EVM node, 0.0 otherwise                                           | Observer, RPC       |
```

Comment:
```suggestion
| bootstrapping                  | gauge     | 1.0 if the EVM node is caught up with its upstream EVM node or 0.0 if it is in the process of bootstrapping                                           | Observer, RPC       |
```
Is this "caught up with?" Seems that I get a 1.0 when my node is bootstrapped.

Reply:
1 means bootstrapping, 0 means up-to-date, there might be an issue if you're node stays at 1 :thinking:

Reply:
Interesting, I guess my node was not bootstrapped when I tested this. I changed my suggestion to match.

Reply:
Suggestion applied